### DISCUSSION TITLE
Neotest with nightly neovim "No tests found" (neotest-python, NVIM v0.11.0-dev-798+ge268fcbda)

### DISCUSSION THREAD

**Discussion by Dronakurl** [LazyVim/LazyVim#4476] [Q&A] [answered]
I want to use Neotest with python. It works fine with nvim stable (0.10), but with the nightly version it says "No tests found".
I tried with a clean installation of lazyvim (so no customization) and I used the LazyExtras to set up python neotest support.
Please, can someone give my  advice on how to narrow it down?

**Comment by dpetka2001** [comment, marked as answer]
It's a problem with neotest itself. Try out this [PR](https://github.com/nvim-neotest/neotest/pull/450) locally and I believe it should work. Test it out to see if that's the case. Or just keep using Neovim stable until it's fixed in neotest for nightly.
